llvm/lib/Target/AMDGPU/AsmParser/AMDGPUAsmParser.cpp

```diff
@@ -5838,10 +5845,14 @@ bool AMDGPUAsmParser::matchAndEmitInstruction(SMLoc IDLoc, unsigned &Opcode,
   MCInst Inst;
   Inst.setLoc(IDLoc);
   unsigned Result = Match_Success;
+  FeatureBitset MissingFeatures;
```

Comment:
Do you need to clear it between loop iterations?

Reply:
Well, the whole PR is not used yet, so it does not matter.

Reply:
I.e., make a use of it and we will see.

Reply:
The point of the PR is so whoever picks this up doesn't have to figure out where in the file to put the .incs